apps/Unciv/install

```diff
@@ -17,34 +17,42 @@ fi
 # so, we will use Temurin OpenJDK 8 installation cause thats the minimal Java version Unciv needs
 # Temurin very smoothly on Raspberry Pi 3B+ without any problems
 
-status "Adding Adoptium repository:"
+if [[ ! -f /usr/share/keyrings/adoptium-archive-keyring.gpg ]]; then
```

Comment:
who said the keyring might not change in the future? I don't really see any benefit to not running the "generate a new keyring" part of the script on every update, it will just make sure that the latest keyring is always installed

Reply:
bruh, you guys are just increasing the update time. treating updates and first time installation is a problem itself. again, when did you last say a public keyring change? never saw it happen for a official repo myself. stop making up unrealistic arguments like kids.

Reply:
doing more thing is just making path for more errors to happen. a download failure is not rare. so no need for downloading things again and again that we do not need.

Reply:
the repo is brand new, it has already gotten a new signing key once (about 1 week ago). You can view the history of this repo addition at the issue relating to it:https://github.com/adoptium/installer/issues/330

I don't expect another key change to occur, but that doesn't mean it won't, which is why I suggested leaving it in on every update/install.
and fyi, the process of downloading, formatting, and adding the key takes well less than a second, so its not increasing the update time in any significant manner

Reply:
reverted the changes in https://github.com/Botspot/pi-apps/pull/1289/commits/1d988707a1a1c0203545b492def38d450f549bb2